tests/UI/commonTests/BO/advancedParameters/setPermissions.js

```diff
@@ -0,0 +1,89 @@
+require('module-alias/register');
+// Import utils
+const helper = require('@utils/helpers');
+
+// Import login test
+const loginCommon = require('@commonTests/BO/loginBO');
+
+// Import BO pages
+const dashboardPage = require('@pages/BO/dashboard');
+const employeesPage = require('@pages/BO/advancedParameters/team/index');
+const permissionsPage = require('@pages/BO/advancedParameters/team/permissions/index');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+// Import expect from chai
+const {expect} = require('chai');
+
+let browserContext;
+let page;
+
+/**
+ *
+ * @param {string} profileName
+ * @param {array} permissionsData
+ * @param {string} baseContext
+ */
```

Comment:
```suggestion
/**
 * Function to set the employee permissions
 * @param profileName {string} 
 * @param permissionsData {array} 
 * @param baseContext {string} 
 */
```
Please add the description of all params

Reply:
Fixed